src/index.html

```diff
@@ -1,18 +1,49 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1.0"
-    />
     <title>Calendar</title>
     <link
       rel="stylesheet"
       href="styles/index.scss"
     />
```

Comment:
Stylesheets with the SCSS extension cannot be linked directly in HTML. You should compile `.scss` files to `.css` and then link the `index.css` file here.